wdl/GATKSVPipelinePhase1.wdl

```diff
@@ -101,6 +101,9 @@ workflow GATKSVPipelinePhase1 {
     Array[File]? wham_vcfs         # Wham VCF
     Int min_svsize                 # Minimum SV length to include
 
+    # PlotSVCountsPerSample metrics
+    Int? N_IQR_cutoff_plotting
```

Comment:
Can you move this input to the ClusterBatch section (around line 162ish) since that is the subworkflow it belongs to? And can you add the runtime attributes as well, at the bottom of the ClusterBatch section with the other runtime attributes? (should be around line 188ish)